.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+## Change description
+
+> Description here
+
+## Type of change
+
+- [ ] Bug fix (fixes an issue)
+- [ ] New feature (adds functionality)
+
+## Related issues
+
+Does this PR fix a Github Issue?
+
+> Fix [#1]()
+
+## Checklists
+
+### Development
+
+- [ ] Application changes have been tested thoroughly
+
+### Security
+
+- [ ] Code follows company security practices and guidelines
+- [ ] Security impact of change has been considered
+  > Potential security impact (if any):
+
+### Code review
+
+- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
+- [ ] "Ready for review" label attached and reviewers assigned
+- [ ] Changes have been reviewed by at least one other contributor
+- [ ] Pull request linked to task tracker where applicable
```

Comment:
This probably isn't needed if we follow our branch naming conventions to start with the Pivotal Tracker story ID